### PR TITLE
ENDOC-476 Update the OpenShift landing page

### PR DESCRIPTION
**Pull request by Lyd1aCla1r3**

### CODE REVIEW

**Review by nshaw — APPROVED** [review]
@Lyd1aCla1r3 I made a couple minor tweaks here. Looks good now.

**Comment by nshaw** [comment]
I converted this to draft to make sure we don't merge it early. :)

File: vuepress/docs/.vuepress/components/LandingPage.vue [hunk 1]
```diff
@@ -51,32 +49,31 @@
...
             <div class="language-sh"><pre>oc get pods -n entando --watch</pre></div>
-            <p>Get the URL to access Entando from your local browser, e.g. <span>quickstart-entando.192.168.64.25.nip.io/app-builder/</span></p>
-            <div class="language-sh"><pre>oc get ingress -n entando -o jsonpath='{.items[2].spec.rules[*].host}{.items[2].spec.rules[*].http.paths[2].path}{"\n"}'</pre></div>
-            <p>Login to the <span>Entando App Builder</span> with username:<span>admin</span>, password: <span>adminadmin</span></p>
-            <p>Choose an action from the <a :href="path('/docs/getting-started/#next-steps')">Next Steps</a> list to continue your journey with Entando!</p>
+            <p>Access Entando from your local browser</p>
+            <div class="language-sh"><pre>http://YOUR-HOST-NAME/app-builder/, e.g. quickstart.YOUR-IP.nip.io/app-builder/</pre></div>
+            <p>Login to the Entando App Builder</p>
+            <div class="language-sh"><pre>username: admin, password: adminadmin</pre></div>
+            <p>Choose an action from <a :href="path('/docs/getting-started/#next-steps')">Next Steps</a> to continue your journey with Entando!</p>
```
Comment (jyunmitch) [inline vuepress/docs/.vuepress/components/LandingPage.vue:76]: does it sound awkward without the or something >> "Choose an action from the Next Steps to continue...."

Comment (Lyd1aCla1r3) [reply vuepress/docs/.vuepress/components/LandingPage.vue:76]: i actually think we should leave out "the"; Next Steps without an article prefacing it is more direct, and there are many incarnations of "next steps," so "the" Next Steps implying that there's only one set of next steps is misleading...

**Comment by Lyd1aCla1r3** [comment]
@nshaw should we change `CTRL+C` to `Ctrl+C`? super minor...

**Pull request merged**